lms/views/api/canvas/sync.py

```diff
@@ -166,6 +173,12 @@ def _get_course(self):
             self._course_service.generate_authority_provided_id(tool_guid, context_id)
         )
 
+    def _get_user(self, user_id):
+        return self._request.find_service(UserService).get(
+            self._request.find_service(name="application_instance").get_current(),
+            user_id,
+        )
```

Comment:
I wonder if this could be:

```python
@property
def user(self):
    ...
```

with no `user_id` argument, since the same value is always passed.

Was also wondering whether we should have `request.user` like we have `request.lti_user`?

Reply:
> Was also wondering whether we should have request.user like we have request.lti_user

Yes I think this is the right call. I have a half baked branch locally where lti_user has a user field. I'll clean that up and push it.